packages/node/src/lib/notification-template/notification-template.interface.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  INotificationTemplateStep,
+  IPreferenceChannels,
+} from 'libs/shared/dist';
+
+import { AxiosPromise } from '.pnpm/axios@0.26.1/node_modules/axios';
+
+export interface INotificationTemplates {
+  create(
+    name: string,
+    notificationGroupId: string,
+    data: INotificationTemplatePayload
+  ): AxiosPromise;
+  update(
+    templateId: string,
+    name: string,
+    data: INotificationTemplatePayload
+  ): AxiosPromise;
+  delete(templateId: string): AxiosPromise;
+  getOne(templateId: string): AxiosPromise;
+  updateStatus(templateId: string, active: boolean): AxiosPromise;
+}
+
+export interface INotificationTemplatePayload {
```

Comment:
Isn't anything missing? Like name, notificationGroupId...

Reply:
That is included in whichever methods required, so I thought not to include in this interface

Reply:
@scopsy wdyt?

Reply:
Yes looks like it's missing here. Sorry for the delayed response, my GitHub notification are going crazy.

@codingmickey would be best to specify the properties from here: https://docs.novu.co/api/create-notification-template/

Reply:
Yes! I got the properties from there.

Oh okay, are you suggesting that I put all the parameters in a single template and then tell the user to put those parameters as the object with `name` and `notificationGroupId`(and not as a different parameter as I'm currently doing it?) as required?

Reply:
Looks great like this!